### PR TITLE
fix: 🐛 Fix the pool tvl display when amount > 1B

### PR DESCRIPTION
**Pull request by chef-ryan**
---

## PR-Codex overview
This PR introduces a new component, `PoolTvlWarning`, to display warnings about TVL and APR data accuracy in the `PoolInfo` view. It also updates localization strings to include a new warning message.

### Detailed summary
- Added `PoolTvlWarning` component in `PoolInfo.tsx`.
- Integrated `PoolTvlWarning` into the `PoolInfo` layout.
- Updated localization in `translations.json` to include:
  - New warning message about TVL and APR data.
  - Added context for users regarding the accuracy of data.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/web/src/views/PoolDetail/components/PoolTvlWarnning.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  }
+  // const pairAddress = Pair.getAddress(token0.wrapped, token1.wrapped)
+  const link = `https://pancakeswap.finance/info/v3/pairs/${id}`
+  return (
+    <Message my="24px" mx="24px" variant="warning">
+      <MessageText fontSize="17px">
+        <Text color="warning" as="span">
+          {t('TVL and APR data may not be reflected correctly due to technical limitation, please refer to')}{' '}
+          <TextLink target="_blank" href={link}>
+            {t('this link')}
```
Comment (chefjackson) [inline apps/web/src/views/PoolDetail/components/PoolTvlWarnning.tsx:23]: 'info page'

File: apps/web/src/views/PoolDetail/components/PoolTvlWarnning.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+const ONE_BILLION = 1_000_000_000
+export const PoolTvlWarning = ({ poolInfo }: { poolInfo: PoolInfo }) => {
+  const { t } = useTranslation()
+  const { id } = useRouterQuery()
+  const tvlUsd = parseFloat(poolInfo.tvlUsd ?? '0')
+  if (tvlUsd < ONE_BILLION) {
+    return null
+  }
+  // const pairAddress = Pair.getAddress(token0.wrapped, token1.wrapped)
+  const link = `https://pancakeswap.finance/info/v3/pairs/${id}`
```
Comment (chefjackson) [inline apps/web/src/views/PoolDetail/components/PoolTvlWarnning.tsx:16]: it could be v2/SS and could be different chain

**Review by chefjackson — CHANGES_REQUESTED** [review]

File: apps/web/src/views/PoolDetail/components/PoolTvlWarnning.tsx (hunk 1 above)
Comment (chef-ryan) [reply apps/web/src/views/PoolDetail/components/PoolTvlWarnning.tsx:23]: done

File: apps/web/src/views/PoolDetail/components/PoolTvlWarnning.tsx (hunk 2 above)
Comment (chef-ryan) [reply apps/web/src/views/PoolDetail/components/PoolTvlWarnning.tsx:16]: done

File: apps/web/src/views/PoolDetail/components/PoolTvlWarnning.tsx
[the remark this answers is not in the document]
Comment (chef-ryan) [reply apps/web/src/views/PoolDetail/components/PoolTvlWarnning.tsx:1]: done

File: apps/web/src/views/PoolDetail/components/PoolTvlWarning.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+export const PoolTvlWarning = ({ poolInfo }: { poolInfo: PoolInfo }) => {
+  const { t } = useTranslation()
+  const { id, chainId } = useRouterQuery()
+  const tvlUsd = parseFloat(poolInfo.tvlUsd ?? '0')
+  const { protocol } = poolInfo
+  const chain = getChainName(chainId)
+
+  const version = `${protocol === Protocol.V3 ? '/v3/' : ''}`
+  const stableSwap = `${protocol === Protocol.STABLE ? '?type=stableSwap' : ''}`
+  const link = `https://pancakeswap.finance/info/${version}${chain}/pairs/${id}${stableSwap}`
```
Comment (chefjackson) [inline apps/web/src/views/PoolDetail/components/PoolTvlWarning.tsx:19]: better use relative link

**Review by chefjackson — APPROVED** [review]

File: apps/web/src/views/PoolDetail/components/PoolTvlWarning.tsx (hunk 3 above)
Comment (chef-ryan) [reply apps/web/src/views/PoolDetail/components/PoolTvlWarning.tsx:19]: done

**Pull request merged**